school/redfish/computersystem.go

```diff
@@ -565,7 +626,7 @@ type MemorySummary struct {
 	Status common.Status
 	// TotalSystemMemoryGiB is the amount of configured system general purpose
 	// volatile (RAM) memory as measured in gibibytes.
-	TotalSystemMemoryGiB int
+	TotalSystemMemoryGiB float32
```

Comment:
Ah, I was wondering if that was going to be an issue. ;)

There are probably more of these cases where we'll have to watch out for places that need updating.